app/controllers/api/feedback_submissions_controller.rb

```diff
@@ -0,0 +1,21 @@
+module Api
+  class FeedbackSubmissionsController < ApiController
+    def create
+      @feedback = FeedbackSubmission.new(feedback_params)
+
+      if @feedback.save
+        ZendeskTicketsJob.perform_later(@feedback)
+      else
+        render status: 422, json: { messages: @feedback.errors.full_messages }
```

Comment:
@alan I'm not suggesting creating 2 calls (using a separate one for validation) in the general case. I'm suggesting that the client does the validation wherever possible. I agree that if a validation error prevents saving an error should be returned, but I'm suggesting that that response doesn't need to be sufficiently rich to render an error page, and should be allowed to trigger a 500 response in pvb-public (that's what I mean by 'error state').

The validation APIs that do exist are special cases since they need to talk to NOMIS, and would be impossible to implement in the client.

I didn't consider using json-api. First impressions – pretty sensible, but potentially a little heavy (although a lot of the spec is optional). Happy to discuss!